api/controllers/usersController.js

```diff
@@ -0,0 +1,52 @@
+const userModel = require('../models/usersModel');
+const bcrypt = require('bcrypt');
+
+const jwt = require('jsonwebtoken');
+
+
+function create(req, res, next) {
+    userModel.create({
+        name: req.body.name,
+        email: req.body.email,
+        password: req.body.password }, (err) => {
+        if (err) {
+            next(err);
+        }
+        else {
+            res.json({
+                status: 'success',
+                message: 'User added successfully!!!',
+                data: null });
+        }
+    });
+}
+
+function authenticate(req, res, next) {
+    userModel.findOne({ email: req.body.email }, (err, userInfo) => {
+        if (err) {
+            next(err);
+        }
+        else {
+            if (bcrypt.compareSync(req.body.password, userInfo.password)) {
+                const token = jwt.sign({ id: userInfo._id }, req.app.get('secretKey'), { expiresIn: '24h' });
```

Comment:
* In my opinion, the expires value is a candidate to be changed recurrently so... (see next point);
* It may be out of scope for this phase, but I suggest you add a comment stating that in future you will make the expire value configurable, i.e. it will be possible to configure that value outside the code;